### PR TITLE
Added Server 2019

### PR DESCRIPTION
**Pull request by joinimran**
This is already supported in Windows Server 2019 and it was missing in the applied to the section. Added this.

Problem: https://github.com/MicrosoftDocs/windows-itpro-docs/issues/8205

### CODE REVIEW

**Comment by joinimran** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/security/threat-protection/windows-firewall/windows-firewall-with-advanced-security.md [hunk 1]
```diff
@@ -22,6 +22,7 @@ ms.custom: asr
 **Applies to**
 -   Windows 10
 -   Windows Server 2016
+-   Windows Server 2019
```
Comment (illfated) [inline windows/security/threat-protection/windows-firewall/windows-firewall-with-advanced-security.md:25]: Codestyle: MarkDown bullet lists only need 1 single space between the bullet marker (hyphen) and its text:
```suggestion
- Windows 10
- Windows Server 2016
- Windows Server 2019
```

**Review by JohanFreelancer9 — APPROVED** [review]
Agree with illfated's suggestion.

**Comment by joinimran** [comment]
Suggestions applied. Interestingly, I have just copied and pasted and changed 6 to 9. That means it was bydefault wrong.

**Review by illfated — APPROVED** [review]
Looks good to me.

**Comment by illfated** [comment]
Thank you for providing the PR. Interesting note about your observed change. Some browsers might display unexpected changes, depending on how well they read GitHub pages.

**Comment by illfated** [comment]
#sign-off

**Review by denisebmsft — APPROVED** [review]

**Comment by denisebmsft** [comment]
Thank you @joinimran

**Pull request merged**